docs/data/material/components/buttons/buttons.md

```diff
@@ -210,4 +228,22 @@ import Button from '@mui/material-next/Button';
 
```

Comment:
Would it make more sense? We already know it's a button page, and "Material" doesn't exists by itself.

"Material 3 Button" -> "Material Design v3"

Reply:
> ...and "Material" doesn't exists by itself

The official Material Design docs do have plenty of mentions of "Material 3", using "Material" standalone like that. [See the homepage](https://m3.material.io/) — "M3", "Material talks", "Material blog", etc.

Reply:
But is it more as a shorthand? Using the full version could avoid confusing. Is Material v3 the same thing as Material UI v3?

Reply:
Heh, the "Is Material as in Material Design or as in Material UI?" question is _the_ question 😬 Anyway, no hard preference on using Material Design v3 — my take was to have a matching terminology with the official MD docs, but it may be worth the clarification.

Reply:
Oh, very nice link (it talks about the default color, too, huh!). I'd say that using "Material Design" to differentiate from Material UI is probably the safest approach most of the time, not just when we initially refer to it.